Automattic/wp-calypso
# Design Picker: Introduce DeviceSwitcher component

#### Proposed Changes

Referring to this triage [PR](https://github.com/Automattic/wp-calypso/pull/70063), we need a `DeviceSwitcher` component so that we can re-use it to render the large preview of the pattern assembler.

As a result, this PR is focusing on making a `DeviceSwitcher` component and we can reuse it anywhere.

| Before | After |
| - | - |
| ![image](https://user-images.githubusercontent.com/13596067/208072878-b95746db-392e-404f-8900-1374e1634c49.png) | ![image](https://user-images.githubusercontent.com/13596067/208072765-7257f8db-6671-4297-ae0b-70943416ff83.png) |
| ![image](https://user-images.githubusercontent.com/13596067/208072538-c9d1e454-5e02-4c59-aa0e-0041928846c1.png) | ![image](https://user-images.githubusercontent.com/13596067/208072439-2f6e9312-59ed-4630-9e4e-63bde7c935b1.png) |
| ![image](https://user-images.githubusercontent.com/13596067/208073054-2e964ef4-85ae-4d61-be8c-44cee0530865.png) | ![image](https://user-images.githubusercontent.com/13596067/208073013-7392c50e-803c-4d00-9d2d-b293b3cff9e4.png) |
| ![image](https://user-images.githubusercontent.com/13596067/208073143-d892e702-013d-4eb8-9538-d880ce3d3690.png) | ![image](https://user-images.githubusercontent.com/13596067/208073183-b6852749-6366-4afa-b2f9-9f457992ee3a.png) |

#### Testing Instructions



* Go to `/setup?siteSlug=<your_site>`
* Click the "Continue" button until you land on the design picker
* Select a theme without any style variation, and verify the preview looks the same as before
* Select a theme with style variations, and verify the preview looks the same as before
* Scroll to the bottom and the generated designs should look the same as before
* Select any generated designs, and verify the preview looks the same as before

#### Pre-merge Checklist





Related to https://github.com/Automattic/wp-calypso/pull/70063, pbxlJb-39t-p2
